<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtStackedWidgetLabel</class>
 <widget class="QWidget" name="QtStackedWidgetLabel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="m_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_prev_button">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources/qtils/stacked_widget/stacked_widget.qrc">
       <normaloff>:/qtils/stacked_widget/images/16x16/left_arrow.png</normaloff>:/qtils/stacked_widget/images/16x16/left_arrow.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_next_button">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../resources/qtils/stacked_widget/stacked_widget.qrc">
       <normaloff>:/qtils/stacked_widget/images/16x16/right_arrow.png</normaloff>:/qtils/stacked_widget/images/16x16/right_arrow.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/qtils/stacked_widget/stacked_widget.qrc"/>
 </resources>
 <connections/>
</ui>
